Setup/UpgradeSchema.php

```diff
@@ -103,7 +105,7 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
                 \Magento\Framework\DB\Ddl\Table::TYPE_INTEGER,
                 null,
                 ['unsigned' => true, 'nullable' => true],
-                'Store ID'
+                'Store Id'
```

Comment:
is this the only place that we need to change the name from ID to Id? no other place in the code should know on this change?

Reply:
It has 0 effect on functionality, it's just the comment for the db column.